.gitignore

```diff
@@ -2,3 +2,4 @@
 /docs/vendor
 .idea
 .*.swp
+/composer.lock
```

Comment:
You can actually remove the `/`

Reply:
Yes, as `vendor` is in gitignore anyway and we don't have any other `composer.lock` in this project, so it is fine.

Reply:
If you're sure, you're sure, I pushed a new version :)